mycroft/audio/audioservice.py

```diff
@@ -496,4 +509,3 @@ def shutdown(self):
         self.bus.remove('recognizer_loop:audio_output_end',
                         self._restore_volume)
         self.bus.remove('recognizer_loop:record_end', self._restore_volume)
-        self.bus.remove('mycroft.stop', self._stop)
```

Comment:
Any reason for this change?

Reply:
Yes, it's removed but it's never added so it's sort of redundant (I put it in a separate commit to differentiate it from the other change)